specs/Deployment/DeploymentAPI.md

```diff
@@ -6,9 +6,9 @@ This is the spec for proposal
 The Windows App SDK (WinAppSDK) for packaged apps involves the framework, main, and singleton
 packages, which are all signed and published by Microsoft. The framework package contains the
 WinAppSDK binaries used at run time, and it is installed with the application. The main package
-contains out-of-process services that are brokered between apps, such as push notifications, and it
-is also needed for the framework to be serviced by the Microsoft Store. The singleton package
-supports a single long-running process that is brokered between apps for features like push
+contains out-of-process services that are brokered between apps, such as Dynamic Dependency data
+store, and it is also needed for the framework to be serviced by the Microsoft Store. The singleton
+package supports a single long-running process that is brokered between apps for features like push
 notifications.
```

Comment:
SUGGESTION: Remove the text about what the Main+Singleton do / why they exist, as that ages (badly). Doesn't really matter here what they do, just that they're needed, and thus DeploymentManager is too